python-net/advanced-operations/working-with-documents/formatting-pdf-document/_index.md

```diff
@@ -0,0 +1,386 @@
+
+    # Specify whether to resize the document's window to fit the size of
+    # First displayed page - Default: false
+    document.fit_window = True
+
+    # Specify whether to hide menu bar of the viewer application - Default: false
+    document.hide_menubar = True
+
+    # Specify whether to hide tool bar of the viewer application - Default: false
+    document.hide_tool_bar = True
+
+    # Specify whether to hide UI elements like scroll bars
+    # And leaving only the page contents displayed - Default: false
+    document.hide_window_ui = True
+
+    # Document's page mode. specify how to display document on exiting full-screen mode.
+    document.non_full_screen_page_mode = ap.PageMode.USE_OC
+
+    # Specify the page layout i.e. single page, one column
+    document.page_layout = ap.PageLayout.TWO_COLUMN_LEFT
+
+    # Specify how the document should display when opened
+    # I.e. show thumbnails, full-screen, show attachment panel
+    document.page_mode = ap.PageMode.USE_THUMBS
+
+    # Save updated PDF file
+    document.save(output_pdf)
+```
+
+### Embedding Standard Type 1 Fonts
+
+Some PDF documents have fonts from a special Adobe font set. Fonts from this set are called “Standard Type 1 Fonts”. This set includes 14 fonts and embedding this type of fonts requires using of special flags i.e [Aspose.Pdf.Document.EmbedStandardFonts](https://reference.aspose.com/pdf/python-net/aspose.pdf/document/#properties). Following is the code snippet which can be used to get a document with all fonts embedded including Standard Type 1 Fonts:
+
+```python
+
+    import aspose.pdf as ap
+
+    # Load an existing PDF Document
+    document = ap.Document(input_pdf)
+    # Set EmbedStandardFonts property of document
+    document.embed_standard_fonts = True
+    for page in document.pages:
+        if page.resources.fonts != None:
+            for page_font in page.resources.fonts:
+                # Check if font is already embedded
+                if not page_font.is_embedded:
+                    page_font.is_embedded = True
+
+    document.save(output_pdf)
+```
+
+### Embedding Fonts while creating PDF
+
+If you need to use any font other than the 14 core fonts supported by Adobe Reader, you must embed the font description while generating the PDF file. If font information is not embedded, Adobe Reader will take it from the Operating System if it’s installed over the system, or it will construct a substitute font according to the font descriptor in the PDF.
+
+>Please note the embedded font must be installed on the host machine i.e. in case of the following code ‘Univers Condensed’ font is installed over the system.
+
+We use the property IsEmbedded of Font class to embed the font information into PDF file. Setting the value of this property to ‘True’ will embed the complete font file into the PDF, knowing the fact that it will increase the PDF file size. Following is the code snippet that can be used to embed the font information into PDF.
```

Comment:
```suggestion
We use the property IsEmbedded of Font class to embed the font information into a PDF file. Setting the value of this property to `True` will embed the complete font file into the PDF, knowing the fact that it will increase the PDF file size. Following is the code snippet that can be used to embed the font information into PDF.
```